transport/src/main/java/io/netty/channel/SingleThreadEventLoop.java

```diff
@@ -27,12 +28,24 @@
  */
 public abstract class SingleThreadEventLoop extends SingleThreadEventExecutor implements EventLoop {
 
+    private final Queue<Runnable> tailTasks;
+    private boolean lastRunTimedOut;
+    private long runTimeout;
+    private final Action<Long> runTimeoutUpdater = new Action<Long>() {
```

Comment:
move this to the other final field so stuff is grouped together in respect to visibility.